src/analysis/locally_convex/basic.lean

```diff
@@ -193,6 +212,9 @@ section normed_field
 variables [normed_field 𝕜] [normed_ring 𝕝] [normed_space 𝕜 𝕝] [add_comm_group E] [module 𝕜 E]
   [smul_with_zero 𝕝 E] [is_scalar_tower 𝕜 𝕝 E] {s t u v A B : set E} {a b : 𝕜}
 
+lemma balanced.symmetric (h : balanced 𝕜 A) {x : E} (hx : x ∈ A) : -x ∈ A :=
```

Comment:
```suggestion
lemma balanced.neg_mem (h : balanced 𝕜 A) {x : E} (hx : x ∈ A) : -x ∈ A :=
```
maybe?

Can you also provide the equjivalence and prove it from `balanced.mem_smul_iff`?